tests/solids/diamondC_1x1x1_pp/CMakeLists.txt

```diff
@@ -463,6 +463,29 @@ if(add_estimator_tests)
   #   check_stats.py -s 0 -q spindensity -e 20 -c 8 -p qmc_spindens_short -r qmc-ref/qmc_spindens_short.s000.stat_ref_spindensity.dat
   #   )
 
+  # # Unlikey check_states.py will be able to digest this
```

Comment:
Unlikey is a typo and this message doesn't help me understand what is unique about this test.